File: src/Android/Projects/Recipes/007-UncaughtExceptionAppApplication/app/src/main/java/org/csystem/samples/uncaughtexceptionapp/SecondActivity.java
package org.csystem.samples.uncaughtexceptionapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);
    }

    public void onOKButtonClicked(View view)
    {
        throw new RuntimeException();
    }
}

File: src/Android/Projects/014-OpenOtherActivity/app/src/main/java/org/csystem/samples/application/openotheractivity/SecondActivity.kt
package org.csystem.samples.application.openotheractivity

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class SecondActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
    }

    fun onOpenThirdActivityClicked(view: View)
    {
        val intent = Intent(this, ThirdActivity::class.java)

        startActivity(intent)
    }
}